### DISCUSSION TITLE
What am I?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#736] [Galaxy Trivia] [answered]
I have more than 3400 distinct species and am considered "annual" but some of my species only emerge from underground once every 17 years.

**Comment by peterbaccaro** [comment, marked as answer]
You are a **periodical cicada**.
A cicada is a type of insect known for its distinct and loud buzzing sound. Cicadas are annual insects, meaning that they emerge from the ground once a year, but there are certain species of cicadas, known as **periodical cicadas**, that only emerge every 13 or 17 years. These cicadas spend most of their life underground as nymphs, feeding on the sap of tree roots. When the time is right, they emerge in large numbers, shed their exoskeleton, and mate. The periodical cicada emergence is a remarkable natural phenomenon and is often referred to as a "cicada invasion" or "cicada outbreak."

**Comment by thuongtruong1009** [comment]
Based on the information provided, you are referring to the insect known as the "Cicada." Cicadas are known for their distinct buzzing sound and are found in many parts of the world. They are considered "annual" because different species of cicadas emerge every year. However, certain species of cicadas, known as periodical cicadas, have an incredibly long life cycle of 13 or 17 years. These periodical cicadas spend the majority of their lives underground as nymphs and emerge in large numbers once every 13 or 17 years to mate and lay eggs.

**Comment by SANKARAMDAS** [comment]
Cicadas.
cicada, (family Cicadidae), any of a group of sound-producing insects (order Homoptera) that have two pairs of membranous wings, prominent compound eyes, and three simple eyes (ocelli). Cicadas are medium to large in size, ranging from 2 to 5 cm (0.8 to 2 inches).
More than 3,000 species of cicadas are known. With the exception of two species of hairy cicadas in the family Tettigaretidae that are found only in southeastern Australia, including Tasmania, cicadas belong to the family Cicadidae and are tropical and occur in deserts, grasslands, and forest.
Among the most fascinating and best-known are the 17-year cicada often erroneously called the 17-year locust and the 13-year cicada Magicicada.
